test/e2e/common.go

```diff
@@ -64,6 +65,56 @@ func dumpSpecResourcesAndCleanup(ctx context.Context, specName string, clusterPr
 	}
 }
 
+const (
+	artifactoryURL = "https://artifactory.nordix.org/artifactory/metal3/images/k8s"
+	imagesURL      = "http://172.22.0.1/images"
+	ironicImageDir = "/opt/metal3-dev-env/ironic/html/images"
+)
+
+func downloadImage(k8sVersion string) (imageURL string, imageChecksum string) {
+	osType := strings.ToLower(os.Getenv("OS"))
+	Expect(osType).ToNot(Equal(""))
```

Comment:
Maybe better to make this explicit
```suggestion
	Expect(osType).To(BeElementOf([]string{"ubuntu", "centos"}))
```

Reply:
done